qa/integration-tests/src/test/java/io/camunda/it/rdbms/db/decisiondefinition/DecisionDefinitionIT.java

```diff
@@ -0,0 +1,224 @@
+      final CamundaRdbmsTestApplication testApplication) {
+    final RdbmsService rdbmsService = testApplication.getRdbmsService();
+    final RdbmsWriter rdbmsWriter = rdbmsService.createWriter(PARTITION_ID);
+    final DecisionDefinitionReader decisionDefinitionReader =
+        rdbmsService.getDecisionDefinitionReader();
+
+    final var decisionDefinition =
+        DecisionDefinitionFixtures.createRandomized(
+            b -> b.decisionDefinitionId("test-process-unique"));
+    createAndSaveDecisionDefinition(rdbmsWriter, decisionDefinition);
+
+    final var searchResult =
+        decisionDefinitionReader.search(
+            new DecisionDefinitionQuery(
+                new DecisionDefinitionFilter.Builder()
+                    .decisionDefinitionIds("test-process-unique")
+                    .build(),
+                DecisionDefinitionSort.of(b -> b),
+                SearchQueryPage.of(b -> b.from(0).size(10))));
+
+    assertThat(searchResult).isNotNull();
+    assertThat(searchResult.total()).isEqualTo(1);
+    assertThat(searchResult.items()).hasSize(1);
+
+    final var instance = searchResult.items().getFirst();
+
+    assertThat(instance.decisionDefinitionKey())
+        .isEqualTo(decisionDefinition.decisionDefinitionKey());
+    assertThat(instance.version()).isEqualTo(decisionDefinition.version());
+    assertThat(instance.name()).isEqualTo(decisionDefinition.name());
+    assertThat(instance.decisionDefinitionId())
+        .isEqualTo(decisionDefinition.decisionDefinitionId());
+    assertThat(instance.decisionRequirementsId())
+        .isEqualTo(decisionDefinition.decisionRequirementsId());
+    assertThat(instance.decisionRequirementsKey())
+        .isEqualTo(decisionDefinition.decisionRequirementsKey());
+  }
+
+  @TestTemplate
+  public void shouldFindAllDecisionDefinitionPaged(
+      final CamundaRdbmsTestApplication testApplication) {
+    final RdbmsService rdbmsService = testApplication.getRdbmsService();
+    final RdbmsWriter rdbmsWriter = rdbmsService.createWriter(PARTITION_ID);
+    final DecisionDefinitionReader decisionDefinitionReader =
+        rdbmsService.getDecisionDefinitionReader();
+
+    final String decisionDefinitionId = DecisionDefinitionFixtures.nextStringId();
+    createAndSaveRandomDecisionDefinitions(
+        rdbmsWriter, b -> b.decisionDefinitionId(decisionDefinitionId));
+
+    final var searchResult =
+        decisionDefinitionReader.search(
+            new DecisionDefinitionQuery(
+                new DecisionDefinitionFilter.Builder()
+                    .decisionDefinitionIds(decisionDefinitionId)
+                    .build(),
+                DecisionDefinitionSort.of(b -> b),
+                SearchQueryPage.of(b -> b.from(0).size(5))));
+
+    assertThat(searchResult).isNotNull();
+    assertThat(searchResult.total()).isEqualTo(20);
+    assertThat(searchResult.items()).hasSize(5);
+  }
+
+  @TestTemplate
+  public void shouldFindAllProcessInstancePageValuesAreNull(
```

Comment:
🔧 Other methods too. Alternatively, you could not mention the entity in the test name and then it's easier to copy stuff (`shouldFindAllWhenPageValuesAreNull`) :)

```suggestion
  public void shouldFindAllDecisionDefinitionPageValuesAreNull(
```

Reply:
Removed the entity name, was easier 😄